browser/flagr-ui/src/components/Flags.vue

```diff
@@ -11,19 +11,26 @@
         <div v-if="loaded">
           <el-row>
             <el-col>
-              <el-input
-                placeholder="Specific new flag description"
-                v-model="newFlag.description">
+              <el-input placeholder="Specific new flag description" v-model="newFlag.description">
```

Comment:
lots of changes in here from prettier, but most of this is just the onCommandDropdown handler and related elements